README.md

```diff
@@ -17,7 +17,7 @@ CrashMonkey also makes use of common Linux file system checker and repair progra
 ### Getting Setup ###
 
 #### Setting Up a VM ####
-The easiest (and recommended) way to start working on (or using) CrashMonkey is to setup a virtual machine and run everything in the VM. This is partly so that any bugs in the kernel module don't bring down your whole system and partly because I just find it easier. In the future I may try to get a Docker running with all the needed packages and files so that things are easy to setup and get running. In the meantime, you should spin up an Ubuntu 14.04 LTS or Ubuntu 16.04.2 LTS VM and work on there. **CrashMonkey is known to work on kernel versions 3.13.0-121-generic and 4.4.0-62-generic.** The Ubuntu VM that you create will also need the following packages to properly build and run CrashMonkey:
+The easiest (and recommended) way to start working on (or using) CrashMonkey is to setup a virtual machine and run everything in the VM. This is partly so that any bugs in the kernel module don't bring down your whole system and partly because I just find it easier. In the future I may try to get a Docker running with all the needed packages and files so that things are easy to setup and get running. In the meantime, you should spin up an Ubuntu 14.04 LTS or Ubuntu 16.04.2 LTS VM and work on there. **CrashMonkey is known to work on kernel versions 3.13.0-121-generic, 4.4.0-62-generic, 4.15.0-041500-generic, and 4.16.0-041600rc7-generic.** The Ubuntu VM that you create will also need the following packages to properly build and run CrashMonkey:
 
 * make
 * git
```

Comment:
Do we also want to state that btrfs-tools, f2fs-tools and xfsprogs have to be installed to run CM on the respective filesystems?